src/vde_backend/preprocessFluksoSensors.py

```diff
@@ -380,7 +377,8 @@ def main():
 
 	# then, compare new config with previous configs and recompute data if necessary
 	print("> Recompute previous data... ")
```

Comment:
The print statement could be after the if condition to avoid printing an action that does not happen.